RevenueCatUI/PaywallView.swift

```diff
@@ -26,9 +26,12 @@ public struct PaywallView: View {
 
     private let mode: PaywallViewMode
     private let fonts: PaywallFontProvider
+    private let displayCloseButton: Bool
```

Comment:
I used `shouldDisplayDismissButton` as the naming for this in [Android](https://github.com/RevenueCat/purchases-android/pull/1390)... Honestly I could go either way, but I think ideally we have the same naming.

Lmk if you prefer this and I can update it in Android

Reply:
Naming conventions in Swift and Kotlin are different, I don't think we can strive to maintain 100% API parity while keeping our APIs "native".
For reference, I checked and there's 0 methods of `SwiftUI.View` that contain the word "should".